.github/workflows/sdk-jobs-pipelines-1c_pipeline_with_hyperparameter_sweep.yml

```diff
@@ -0,0 +1,47 @@
+name: sdk-jobs-pipelines-1e_pipeline_with_registered_components
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 */8 * * *"
+  pull_request:
+    branches:
+      - sdk-preview
+      - march-sdk-preview
+      - pipeline/**
+    paths:
+      - sdk/**
+      - .github/workflows/sdk-jobs-pipelines-1c_pipeline_with_hyperparameter_sweep.yml
+      - notebooks/dev-requirements.txt
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - name: check out repo
+      uses: actions/checkout@v2
+    - name: setup python
+      uses: actions/setup-python@v2
+      with: 
+        python-version: "3.8"
+    - name: pip install notebook reqs
+      run: pip install -r notebooks/dev-requirements.txt
+    - name: azure login
+      uses: azure/login@v1
+      with:
+        creds: ${{secrets.AZ_AE_CREDS}}
+    - name: setup SDK
+      run: bash setup.sh
+      working-directory: sdk
+      continue-on-error: true
+    - name: setup CLI
+      run: bash setup.sh
+      working-directory: cli
+      continue-on-error: true
+    - name: run jobs/pipelines/1c_pipeline_with_hyperparameter_sweep/pipeline_with_hyperparameter_sweep.ipynb
```

Comment:
Note that current ci won't wait the experiment to run till completed. Instead it will finish right after the job is submitted successfully. Should we  change the logic? @wangchao1230

Reply:
Let's do a common change later.